src/gui/components/tabs/tabs.styles.js

```diff
@@ -0,0 +1,8 @@
+import { css } from 'glamor'
+
+const layout = css({
+  position: 'relative',
+  border: '1px solid #ddd',
```

Comment:
ok I understand, but how do you compute the value in em of your border size in your component ? you but 0.1em because it seems low ?